### ISSUE TITLE
TODO on Compiling page

### ISSUE THREAD

**Issue by olevett** [reactiveui/website#85]
Compiling page currently has a TODO for "provide overview of what each project does; document magic tricks" - we should totally do this, but I think an issue is a slightly clearer place to flag it up.

https://reactiveui.net/docs/getting-started/compiling/

I think the things worth flagging right now are
- What happens with ReactiveUI-Events and how it gets built
- Running unit tests
- What the split between the TargetPlatforms and completely separate packages (e.g. ReactiveUI.XamForms) are and why they have to be like that.

**Comment by glennawatson** [comment]
We should probably move this compiling page into the contributor section of the website maybe?

**Comment by olevett** [comment]
Agreed, it's definitely not a "as a consumer of RxUI you should know..."

**Comment by glennawatson** [comment]
Thanks - closing as superseded. The /docs/getting-started/compiling/ page no longer exists in the current site. Contributor-facing build guidance lives in the reactiveui/ReactiveUI repo's CONTRIBUTING.md.
